book_service/views.py

```diff
@@ -20,32 +21,14 @@ def get_permissions(self):
             return [permissions.IsAdminUser()]
 
         elif self.action == "list":
-            return [permissions.IsAuthenticatedOrReadOnly()]
+            return [permissions.AllowAny()]
 
         elif self.action == "retrieve":
             return [permissions.IsAuthenticated()]
 
         return super().get_permissions()
 
-    @extend_schema(
-        parameters=[
-            OpenApiParameter(
-                name="search",
-                description=(
-                        "Search books by title or author,"
-                        " ?search=title/author"
-                ),
-                type=str,
-                required=False,
-            ),
-            OpenApiParameter(
-                name="ordering",
-                description="Order books by author ?ordering=author",
-                type=str,
-                required=False,
-            ),
-        ]
-    )
+    @extend_schema_view(list=book_list_schema)

```

Comment:
You have to decorate your BookVieiwSet class with it

Reply:
fixed in last commit